contentcuration/contentcuration/frontend/channelEdit/views/ResourcePanel.vue

```diff
@@ -472,23 +472,28 @@
       /* VALIDATION */
       // License isn't specified
       noLicense() {
-        return !this.isTopic && validateNodeLicense(this.node).length;
+        return Boolean(!this.isTopic && validateNodeLicense(this.node).length);
       },
       // Copyright holder isn't set on non-public domain licenses
       noCopyrightHolder() {
-        return !this.isTopic && !validateNodeCopyrightHolder(this.node).length;
+        return Boolean(!this.isTopic && validateNodeCopyrightHolder(this.node).length);
       },
       // License description isn't provided on special permissions licenses
       noLicenseDescription() {
-        return !this.isTopic && !validateNodeLicenseDescription(this.node).length;
+        return Boolean(
+          this.license &&
```

Comment:
Similarly to the comment below regarding `is_custom` check, this check is already present in `validateNodeLicenseDescription`. I would prefer to leave it there, because it's using [this function to retrieve a license in a correct way](https://github.com/learningequality/studio/blob/develop/contentcuration/contentcuration/frontend/shared/utils/validation.js#L65) - again, this is one of the steps towards better validation consistency that will be important on frontend now when it's the only entity responsible for marking a node as `complete`.

Reply:
Okay great - I'm really not sure why I felt that this made it work but clearly I wasn't fully grokking the scope of the code. Thanks @MisRob